### ISSUE TITLE
updatewcs

### ISSUE THREAD

**Issue by amn3142** [spacetelescope/stwcs#28]
Just installed the most recent astroconda build. This includes stwcs version 1.2.5, running with python 2.7.13. In the python terminal I try to run the following:

```python
     import stwcs
    from stwcs import updatewcs
    updatewcs.updatewcs(['fltname.fits'])
```

I get the following error:
``` 
LookupError: 
Problem finding row in IDCTAB! Could not find row matching:
        CHIP: 1
     FILTERS: F140W 
```

I have checked that my $iref and $jref values are set correctly and that the IDCTAB table listed in my flt header is in the $iref directory. 

Note that this problem does not occur when running the version with my old ureka installation, which had stwcs Version: 1.2.1.dev39127. With this older version I get the following output after running the same code as above:
    - IDCTAB: Distortion model from row 4 for chip 1 : F140W
    - IDCTAB: Distortion model from row 4 for chip 1 : F140W
    ['fltname.fits']

Which I assume means that everything worked.

**Comment by nden** [comment]
@amn3142  Could you post your file somewhere or at least its name so I can retrieve it from the archive.

**Comment by amn3142** [comment]
@nden here is the link to the set of observations: [WFI2026](http://archive.stsci.edu/cgi-bin/mastpreview?mission=hst&dataid=ICO205010) using ico205lnq_flt.fits in the above example.

**Comment by nden** [comment]
@amn3142 I retrieved the file and ran it through stwcs without problems. I wonder if this is a setup problem. Could you make sure you are really running stwcs from anaconda
```
>>> updatewcs.__file__
```

should point to the anaconda installation directory.
The idctab I am using is `iref$w3m18525i_idc.fits`. Please confirm this is what you have as well.

Is this the only file that you have trouble with?

**Comment by amn3142** [comment]
Aha! So I verified that stwcs was running from astroconda, but while I was at it I decided to check the other dependencies. It turns out astropy was running from ```~/.local```. Once I removed this version, ```pip show``` points to the astroconda version and now updatewcs works! 

I have no idea how that ```~/.local ```version of astropy got there, or why that's where astropy was running from. 
Thank you so much for your help!
